cmd/sensorbee/default_main.go

```diff
@@ -20,8 +21,8 @@ func init() {
 func main() {
 	app := cli.NewApp()
 	app.Name = "sensorbee"
-	app.Usage = "SensorBee"
-	app.Version = "0.3.2" // TODO: don't hardcode the version number
+	app.Usage = "SensorBee built with build_sensorbee 0.4.1"
```

Comment:
Is `build_sensorbee` command version is always linked to sensorbee revision? I think this usage phrase is also used `version.Version`, like

```
fmt.Sprintf("SensorBee build with build_sensorbee %s", version.Version)
```